modules/standard/Types.chpl

```diff
@@ -332,6 +338,10 @@ proc isUnmanagedClass(type t) param  return isUnmanagedClassType(t);
 pragma "no doc"
 proc isBorrowedClass(type t) param  return isBorrowedClassType(t);
 pragma "no doc"
+proc isNilableClass(type t) param  return isNilableClassType(t);
+pragma "no doc"
+proc isNonNilableClass(type t) param  return isNonNilableClassType(t);
```

Comment:
I see `isNonNilableClassType` and `isNilableClassType` used here but I don't see it exposed in the documentation yet.  We have various other `is*Type` functions exposed [here](https://chapel-lang.org/docs/master/modules/standard/Types.html#Types.isPrimitiveType) in addition to the generic argument version, I think it would be good to follow that precedent.

The rest of the API as it stands seems good to me!

Reply:
That's because isNilableClassType, like isClassType, are in ChapelBase. I can try moving them to Types.

Reply:
Ah, okay!  Nah, moving doesn't seem necessary, as long as it is able to be found

Reply:
You could add a link to it from the generic version, but I'm okay with not doing so.

Reply:
There are a lot of isBlaType functions that aren't documented at all right now. Since isBla() works on type or value arguments though, I don't really see a problem with that. Anyway I'm going to leave it alone right now (vs. trying to move them to Types) since there seem to be other problems here. (and if we want to fix them, better to do that in another PR).

Reply:
Okay, then I guess open an issue to track that action item, unless it's going to get solved right away?

Reply:
#13220